cluster-sync/clean.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -e
+  
+source ./hack/build/config.sh
+source ./cluster-up/hack/common.sh
+source ./cluster-up/cluster/${KUBEVIRT_PROVIDER}/provider.sh
+
+echo "Cleaning up ..."
```

Comment:
It seems to be cdi specific. As one of the next steps we can add kubevirt cleanup as well.